mmv1/products/dns/terraform.yaml

```diff
@@ -27,6 +27,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         min_version: beta
         ignore_read_extra:
           - "force_destroy"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_record_set_basic"
+        primary_resource_id: "default"
```

Comment:
I think we're assuming the `primary_resource_id` would be for `google_dns_managed_zone` when we do the import. [This](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6173-old..auto-pr-6173#diff-40524a01cbca31921bea35a20c5fb30f7fae6a968e81337c5d581d71c1108530R130) is what it generates downstream and the cause of the test failure.